recipe/meta.yaml

```diff
@@ -10,27 +10,31 @@ source:
   sha256: be60da6b778cc5a5296a69d8e7d614f1f870faf94e1b1b6ac591f2ad5d729579
 
 build:
-  noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True # [py<36]
 
 requirements:
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
   imports:
     - pyjsparser
+  command:
+    - pip check
 
 about:
   home: "https://github.com/PiotrDabkowski/pyjsparser"
```

Comment:
We might want to remote the quotation marks around the `home url`:

https://github.com/AnacondaRecipes/pyjsparser-feedstock/blob/b8d314d3cdf75d40f3c516b724b39b9f6f5e9894/recipe/meta.yaml#L33

Reply:
removed quotations